### PR TITLE
Feature/support testnet mode

### PR DESCRIPTION
**Pull request by n13**
Support for running on testnet

### CODE REVIEW

File: ios/Podfile.lock [hunk 1]
```diff
@@ -10,18 +10,18 @@ PODS:
   - Firebase/RemoteConfig (10.3.0):
```
Comment (n13) [inline ios/Podfile.lock:10]: I did not modify this

**Comment by gguijarro-c-chwy** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>:exclamation: PR has more than 500 line changes </td>
    </tr>
  </tbody>
</table>



> Pull Request size seems relatively large. If Pull Request contains multiple changes, split each into separate PR will helps faster, easier review.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 58f9c958c7f2ccf8523f8a85c23a9c5ede8672bc
</p>

File: lib/core/network/repository/profile_repository.dart [hunk 2]
```diff
@@ -13,13 +13,19 @@ class ProfileService extends NetworkingManager {
 
   Future<Result<ProfileData, HyphaError>> getProfile(String accountName) async {
     final url = '${_remoteConfigService.profileServiceEndpoint}${Endpoints.pppProfile}/$accountName';
-    final response = await get(url);
-    if (response.statusCode == 200) {
-      final map = Map<String, dynamic>.from(response.data);
-      return Result.value(ProfileData.fromJson(map));
-    } else {
-      print('get profile error: ${response.statusCode} ${response.statusMessage}');
-      return Result.error(HyphaError(type: HyphaErrorType.api, message: 'server error ${response.statusMessage}'));
```
Comment (n13) [inline lib/core/network/repository/profile_repository.dart:22]: the get call throws on a 500, wrapping in try catch

File: lib/ui/onboarding/import_account/usecases/find_account_use_case.dart [hunk 3]
```diff
@@ -30,7 +32,8 @@ class FindAccountsUseCase extends InputUseCase<Result<Iterable<UserProfileData>,
       final List<Result<ProfileData, HyphaError>> profiles = await Future.wait(futures);
 
       final List<UserProfileData> wtf = List.empty(growable: true);
-      for (final response in profiles) {
```
Comment (n13) [inline lib/ui/onboarding/import_account/usecases/find_account_use_case.dart:33]: handle case when there is no profile available

File: pubspec.lock [hunk 4]
```diff
@@ -5,336 +5,384 @@ packages:
     dependency: transitive
     description:
       name: _fe_analyzer_shared
```
Comment (n13) [inline pubspec.lock:7]: IDK why this is happening...

**Comment by n13** [comment]
> Fails
> 🚫	❗ PR has more than 500 line changes
> > Pull Request size seems relatively large. If Pull Request contains multiple changes, split each into separate PR will helps faster, easier review.
> 
> Generated by 🚫 [dangerJS](https://danger.systems/js) against [85fb3fa](https://github.com/hypha-dao/hypha_wallet/commit/85fb3faf137a6d35a6091869194ddd3d39ab5b81)

It keeps changing pubspec.lock - IDK why

File: ios/Podfile.lock (hunk 1 above)
Comment (gguijarro-c-chwy) [reply ios/Podfile.lock:10]: I know this is annoying, not sure why its happening..

File: lib/ui/onboarding/import_account/usecases/find_account_use_case.dart [hunk 5]
```diff
@@ -16,7 +17,7 @@ class FindAccountsUseCase extends InputUseCase<Result<Iterable<UserProfileData>,
   @override
   Future<Result<Iterable<UserProfileData>, HyphaError>> run(String input) async {
     final eosClient = EOSClient(
-      baseUrl: remoteConfigService.pushTransactionNodeUrl(),
+      baseUrl: remoteConfigService.baseUrl(),
```
Comment (gguijarro-c-chwy) [inline lib/ui/onboarding/import_account/usecases/find_account_use_case.dart:20]: when do we use pushTransactionNodeUrl and when baseUrl

File: lib/ui/onboarding/import_account/usecases/find_account_use_case.dart [hunk 6]
```diff
@@ -30,7 +31,8 @@ class FindAccountsUseCase extends InputUseCase<Result<Iterable<UserProfileData>,
       final List<Result<ProfileData, HyphaError>> profiles = await Future.wait(futures);
 
       final List<UserProfileData> wtf = List.empty(growable: true);
```
Comment (gguijarro-c-chwy) [inline lib/ui/onboarding/import_account/usecases/find_account_use_case.dart:33]: lmao i just noticed this variable name. oopss

File: lib/ui/onboarding/import_account/usecases/find_account_use_case.dart (hunk 3 above)
Comment (gguijarro-c-chwy) [reply lib/ui/onboarding/import_account/usecases/find_account_use_case.dart:33]: This is good to know. I didnt know this was possible.

**Review by gguijarro-c-chwy — APPROVED** [review]

File: lib/ui/onboarding/import_account/usecases/find_account_use_case.dart (hunk 3 above)
Comment (n13) [reply lib/ui/onboarding/import_account/usecases/find_account_use_case.dart:33]: Me neither ;)

File: lib/ui/onboarding/import_account/usecases/find_account_use_case.dart (hunk 5 above)
Comment (n13) [reply lib/ui/onboarding/import_account/usecases/find_account_use_case.dart:20]: Always baseURL, except when we want to push a transaction.

So if we use this URL to sign a transaction and send it off, we use pushTransactionNodeURL

The idea is that that URL points to a fast node - like greymass

A lot of nodes don't have all the features though so for all read operations, and search accounts also, we use baseURL which is set to the load balancer, which automatically picks a node that has what we need for the call in question. 

All nodes have normal chain lookups - like loading table data. 

Only some nodes have a history plugin

And only some nodes have the account lookup - not sure if that is part of history plugin. I didn't even know normal nodes don't have this by default until it failed ...

File: lib/ui/onboarding/import_account/usecases/find_account_use_case.dart (hunk 6 above)
Comment (n13) [reply lib/ui/onboarding/import_account/usecases/find_account_use_case.dart:33]: 😂

**Pull request merged**